docs/getting-started.md

```diff
@@ -55,17 +36,38 @@ curl -fLo coursier https://git.io/coursier-cli \
 bitsadmin /transfer downloadCoursierCli https://git.io/coursier-cli "%cd%\coursier"
 bitsadmin /transfer downloadCoursierBat https://git.io/coursier-bat "%cd%\coursier.bat"
 ./coursier launch com.sourcegraph:lsif-java_2.13:@STABLE_VERSION@ -- --help
+
+# Homebrew
+brew install coursier/formulas/coursier \
+ && coursier launch com.sourcegraph:lsif-java_2.13:@STABLE_VERSION@ -- --help
 ```
 
+Additional command-line flags can be passed after the `--` argument. For
+example, replace `--help` with `index` in the command above to run the `index`
+subcommand.
+
 ### Java fat jar
 
-Use the Coursier `bootstrap` command to generate an executable Java binary,
-which includes all dependencies and does not require further access to the
-internet after installation.
+Use the `lsif-java` fat jar if your CI does not allow downloading binaries from
+the internet.
+
+Use the [Coursier](https://get-coursier.io/docs/cli-installation) `bootstrap`
+command to generate a local fat jar binary. The fat jar binary includes all
+dependencies and does not require further access to the internet after
+installation. The local fat jar will somehow need to be made available to your
+CI machine.
```

Comment:
imo we should supply a fat-jar as a downloadable artifact *as well*.

I would imagine that "if your CI does not allow downloading binaries from the internet" holds true (for security reasons), some companies (especially in certain industries) would have an auditing process involved in getting coursier approved first on top of lsif-java itself, resulting in customers who would prefer to just have a downloadable fat-jar to reduce the entry barrier.

Just my 2c after much back-n-forth with some guy online working at a financial institute looking to adopt Golang and the many hoops they have to go through to add anything new to the developer workflow.

Reply:
The instructions to generate the fat-jar are copy-pasteable commands that you can run anywhere to generate the standalone executable. My issue with uploading fat jars to GitHub releases is that it doesn't provide a clean way for users to upgrade between versions. With the coursier bootstrap command the lsif-java version number is part of the copy-pasted command.

Reply:
I can imagine that some customers will still ask to have a downloadable fatjar from GitHub releases and maybe we'll just comply because it's easier. It's not complicated to generate a fatjar in CI and upload it

Reply:
Totally, we can strongly recommend coursier for that reason. As you said, releasing fatjars is trivial enough, especially given we're already using github actions :slightly_smiling_face: